charts/victoria-metrics-cluster/values.yaml

```diff
@@ -70,6 +70,13 @@ vmselect:
       timeoutSeconds: 5
       failureThreshold: 3
 
+# Horizontal Pod Autoscaling
+  horizontalPodAutoscaler:
+    enabled: false
+    maxReplicas: 10
+    minReplicas: 1
+    targetCPUUtilizationPercentage: 70
```

Comment:
can we add section metrics here 
something like

```
metrics: []
```

and user can define own metric here like 
```
metrics:
  - type: Resource
    resource:
        name: cpu
        targetAverageUtilization: 60